frontend/src/components/admin-page/helpers/check-identifier-type.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { CellValue } from 'react-table';
+import { DateIdentifier } from 'common/enums';
+import { Column } from 'common/interfaces';
+
+const checkIdentifierType = (identifier:string):Column =>{
+    return {
+      Header: identifier,
+      accessor: identifier,
+      Cell : ({value}:CellValue)=>{
+        const dates:string[] = [DateIdentifier.BIRTHDATE,DateIdentifier.CREATED_AT,DateIdentifier.UPDATED_AT];
+        if(dates.includes(identifier)){
+          return value.toString().slice(0,10).replaceAll('-', '.')
+        }else{
+          return value ? value.toString() : ''
```

Comment:
Too hard
You can do something like `value instanceof Date` and check if it date
But it also too hard I think

Reply:
`value instanceof Date` will always return false, because all value are a string

Reply:
Just map users from the server